src/shared/components/annotation/OncoKB.spec.tsx

```diff
@@ -4,8 +4,8 @@ import React from 'react';
 import { assert } from 'chai';
 import {shallow, mount, ReactWrapper} from 'enzyme';
 import sinon from 'sinon';
-import {mskTableSort} from "../msktable/MSKTable";
-import {IndicatorQueryResp} from "shared/api/generated/OncoKbAPI";
+import {lazyMobXTableSort} from "../lazyMobXTable/LazyMobXTable";
+import {IndicatorQueryResp} from "../../api/generated/OncoKbAPI";
```

Comment:
Why do we need to change `shared/api/generated/OncoKbAPI` to `../../api/generated/OncoKbAPI`? I prefer the full path to relative path.